spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/TestSparkDistributionAndOrderingUtil.java

```diff
@@ -252,23 +254,8 @@ public void testDefaultWritePartitionedUnsortedTable() {
         };
 
     checkWriteDistributionAndOrdering(table, expectedDistribution, expectedOrdering);
-  }
-
-  @Test
-  public void testDefaultWritePartitionedUnsortedTableFanout() {
```

Comment:
We did have some tests twice: one with fanout writers and one without. I squashed them.